## Estatísticas de Meios de Pagamentos

Este conjunto contém diversas estatísticas relacionadas aos seguintes MEIOS DE PAGAMENTO:

Instrumentos de periodicidade mensal, com atualização 17 dias após o final do mês de referência:

Pix (fonte: SPI e documento 1201)
TED (fonte: CIP-SITRAF e STR)
Boletos (fonte: CIP-SILOC)
DOC e TEC (fonte: CIP-SILOC)
Cheque (fonte: Compe)
Instrumentos de periodicidade trimestral, com atualização 2 meses após o final do trimestre de referência:

Cartões de crédito, débito e pré-pagos (fonte: documentos 6308 e 6334)
Pagamentos de convênio, saques, transferências intrabancárias e débito direto (fonte: documento 6209)
Volumetria das transações intrabancárias (fonte: documento 6209)
Estabelecimentos credenciados (fonte: documento 6334)
Volumetria por canal de acesso e produto (fonte: documento 6209)
Quantidade de ATMs (fonte: documento 6209)
Tarifas de intercâmbio (TIC) e taxas de desconto (MDR) - (fonte: documento 6334)
Terminais POS/PDV
Programas de recompensa e fidelidade

Dicionário de dados

| Nome                         | Tipo    | Título                                    | Descrição                                                                                                                                                                                                                                       |
| ---------------------------- | ------- | ----------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| datatrimestre                | texto   | Trimestre                                 |                                                                                                                                                                                                                                                 |
| valorPix                     | decimal | Valor Pix                                 | Volume financeiro (R\$ milhões) de transações Pix liquidadas trimestralmente no SPI e fora do SPI, considerando ordens de pagamento e devoluções no período. Dados referentes às transações liquidadas fora do SPI estão sujeitas a alterações. |
| valorTED                     | decimal | Valor TED                                 | Montante financeiro (R\$ milhões) trimestral transferido por meio de TED. Transferência Eletrônica Direta (TED) em tempo real entre diferentes instituições, usada para transferir valores entre correntistas e entre instituições.             |
| valorTEC                     | decimal | Valor TEC                                 | Montante financeiro (R\$ milhões) trimestral transferido por meio de TEC. Transferência Especial de Crédito (TEC) utilizada por empresas para pagamento de benefícios aos empregados.                                                           |
| valorCheque                  | decimal | Valor Cheque                              | Montante financeiro (R\$ milhões) de cheques interbancários e intrabancários compensados trimestralmente.                                                                                                                                       |
| valorBoleto                  | decimal | Valor Boleto                              | Montante financeiro (R\$ milhões) de boletos interbancários e intrabancários compensados trimestralmente.                                                                                                                                       |
| valorDOC                     | decimal | Valor DOC                                 | Montante financeiro (R\$ milhões) trimestral transferido por meio de DOC.                                                                                                                                                                       |
| valorCartaoCredito           | decimal | Valor Cartão de Crédito                   | Valor (R\$ milhões) de transações realizadas com cartão de crédito.                                                                                                                                                                             |
| valorCartaoDebito            | decimal | Valor Cartão de Débito                    | Valor (R\$ milhões) de transações realizadas com cartão de débito trimestralmente.                                                                                                                                                              |
| valorCartaoPrePago           | decimal | Valor Cartão Pré-pago                     | Valor (R\$ milhões) de transações realizadas com cartão pré-pago trimestralmente.                                                                                                                                                               |
| valorTransIntrabancaria      | decimal | Valor Transferência Intrabancária         | Montante financeiro (R\$ milhões) de transferências realizadas trimestralmente entre contas de clientes da Instituição, inclusive aplicações e resgates em fundos de investimento.                                                              |
| valorConvenios               | decimal | Valor Convênio                            | Montante financeiro (R\$ milhões) de arrecadações trimestrais governamentais (tributos e encargos sociais) e não-governamentais, em virtude de convênios firmados entre a instituição e entidades públicas ou privadas.                         |
| valorDebitoDireto            | decimal | Valor Débito Direto                       | Montante financeiro (R\$ milhões) trimestral de débitos previamente autorizados pelo cliente, referente a contas recorrentes e tarifas de serviços.                                                                                             |
| valorSaques                  | decimal | Valor Saque                               | Montante sacado (R\$ milhões) nos caixas eletrônicos trimestralmente.                                                                                                                                                                           |
| quantidadePix                | decimal | Quantidade Pix                            | Quantidade (em milhares) de transações Pix liquidadas trimestralmente no SPI e fora do SPI, considerando ordens de pagamento e devoluções.                                                                                                      |
| quantidadeTED                | decimal | Quantidade TED                            | Quantidade (em milhares) de TED realizadas trimestralmente.                                                                                                                                                                                     |
| quantidadeTEC                | decimal | Quantidade TEC                            | Quantidade (em milhares) de TEC realizadas trimestralmente.                                                                                                                                                                                     |
| quantidadeCheque             | decimal | Quantidade Cheque                         | Quantidade (em milhares) de cheques interbancários e intrabancários compensados trimestralmente.                                                                                                                                                |
| quantidadeBoleto             | decimal | Quantidade Boleto                         | Quantidade (em milhares) de boletos interbancários e intrabancários compensados trimestralmente.                                                                                                                                                |
| quantidadeDOC                | decimal | Quantidade DOC                            | Quantidade (em milhares) de DOC realizados trimestralmente.                                                                                                                                                                                     |
| quantidadeCartaoCredito      | decimal | Quantidade Cartão de Crédito              | Quantidade (em milhares) de transações realizadas com cartão de crédito trimestralmente.                                                                                                                                                        |
| quantidadeCartaoDebito       | decimal | Quantidade Cartão de Débito               | Quantidade (em milhares) de transações realizadas com cartão de débito trimestralmente.                                                                                                                                                         |
| quantidadeCartaoPrePago      | decimal | Quantidade Cartão Pré-pago                | Quantidade (em milhares) de transações realizadas com cartão pré-pago trimestralmente.                                                                                                                                                          |
| quantidadeTransIntrabancaria | decimal | Quantidade de Transferência Intrabancária | Quantidade (em milhares) de transferências realizadas trimestralmente entre contas de clientes da Instituição, inclusive aplicações e resgates em fundos de investimento.                                                                       |
| quantidadeConvenios          | decimal | Quantidade Convênio                       | Quantidade (em milhares) de arrecadações trimestrais governamentais e não-governamentais referentes a convênios firmados entre a instituição e entidades públicas ou privadas.                                                                  |
| quantidadeDebitoDireto       | decimal | Quantidade Débito Direto                  | Quantidade (em milhares) de transações trimestrais de débitos previamente autorizados pelo cliente, referentes a contas recorrentes e tarifas de serviços.                                                                                      |
| quantidadeSaques             | decimal | Quantidade de Saque                       | Quantidade (em milhares) de saques realizados nos caixas eletrônicos trimestralmente.                                                                                                                                                           |


## Análise exploratória

In [53]:
import pandas as pd
import sqlite3

con = sqlite3.connect("src/datasets/etlbcb.db")
query = "select * from meios_pagamentos_tri"
df = pd.read_sql(query, con)
con.close()

In [54]:
df

datatrimestre    valorPix     valorTED  valorTEC  valorCheque  \
0   2024-12-31 00:00:00  7714115.37  11647265.67      0.00    187486.45   
1   2024-09-30 00:00:00  6975429.47  11180217.68      0.00    194330.60   
2   2024-06-30 00:00:00  6284296.05  10662555.73      0.00    221553.82   
3   2024-03-31 00:00:00  5429305.43   9651625.56    925.26    190457.25   
4   2023-12-31 00:00:00  5300155.00  10090921.81   5787.89    221424.78   
5   2023-09-30 00:00:00  4469529.57   9960251.89   4156.38    218121.88   
6   2023-06-30 00:00:00  3900377.43  10657463.13   4770.74    228408.82   
7   2023-03-31 00:00:00  3447198.28   9919602.84   4535.17    240686.19   
8   2022-12-31 00:00:00  3342913.94  10382121.63   5609.53    242259.61   
9   2022-09-30 00:00:00  2936413.95  10437763.16   3735.46    267090.22   
10  2022-06-30 00:00:00  2543384.40  10449574.67   4786.14    271008.09   
11  2022-03-31 00:00:00  2067826.55   9507158.34   3865.81    255455.43   
12  2021-12-31 00:00:00  1916418.68   9807293.11   5436.19    264298.16   
13  2021-09-30 00:00:00  1556916.22   9257046.28   3684.13    271120.34   
14  2021-06-30 00:00:00  1105735.20   8566787.27   3894.81    264944.31   
15  2021-03-31 00:00:00   625046.52   7894414.16   4352.44    253527.92   
16  2020-12-31 00:00:00   149894.91   8036705.32   5137.42    278295.37   
17  2020-09-30 00:00:00        0.00   7503982.24   4159.60    259334.88   
18  2020-06-30 00:00:00        0.00   6267004.16   4647.32    237873.12   
19  2020-03-31 00:00:00        0.00   6652135.13   4829.56    318844.92   
20  2019-12-31 00:00:00        0.00   6560042.73   6343.77    372693.66   
21  2019-09-30 00:00:00        0.00   6285701.44   5469.66    385735.21   
22  2019-06-30 00:00:00        0.00   5700739.79   5506.27    388525.47   
23  2019-03-31 00:00:00        0.00   5039380.73   4794.82    375916.28   

    valorBoleto  valorDOC  valorCartaoCredito  valorCartaoDebito  \
0    2530070.72      0.00           728621.99          259929.79   
1    2445001.47      0.00           663288.44          241932.98   
2    2360423.79      0.00           631811.52          239101.56   
3    2248972.68    650.80           607470.96          235111.60   
4    2317765.95   4684.98           630191.34          259057.92   
5    2296686.99  10131.89           581902.42          240827.84   
6    2249059.61  11258.23           561910.94          238172.21   
7    2211522.99   7467.11           535520.44          237175.13   
8    2259674.53   9584.63           560765.41          264120.64   
9    2248416.63  17106.46           521942.27          243632.86   
10   2210099.22  17081.53           501514.74          245563.75   
11   2032317.14  11976.36           453292.41          230186.87   
12   2072053.93  14166.99           467911.25          257458.30   
13   2001074.00  22306.75           399054.11          233970.03   
14   1839452.36  23158.84           344999.68          214381.06   
15   1769759.68  21777.70           310412.13          204752.91   
16   1919100.60  38705.14           332987.17          258471.49   
17   1959819.72  52156.84           279176.98          228771.89   
18   1366623.21  50881.15           224389.54          152246.79   
19   1526486.76  30628.55           275659.09          171399.49   
20   1579497.09  37728.79           302888.19          193673.01   
21   1484836.27  39652.79           272414.67          164735.63   
22   1409618.75  34716.55           253161.38          153747.19   
23   1468066.32  35812.80           235435.49          151764.67   

    valorCartaoPrePago  ...  quantidadeCheque  quantidadeBoleto  \
0             85637.90  ...          41511.28        1459164.03   
1             78531.94  ...          44243.73        1466737.62   
2             77389.01  ...          44981.97        1439100.25   
3             73480.01  ...          46468.88        1422731.42   
4             79592.40  ...          52101.15        1452479.47   
5             69553.22  ...    

Primeiro vamos explorar a estrutura do conjunto de dados

In [55]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 24 entries, 0 to 23
Data columns (total 27 columns):
 #   Column                        Non-Null Count  Dtype  
---  ------                        --------------  -----  
 0   datatrimestre                 24 non-null     object 
 1   valorPix                      24 non-null     float64
 2   valorTED                      24 non-null     float64
 3   valorTEC                      24 non-null     float64
 4   valorCheque                   24 non-null     float64
 5   valorBoleto                   24 non-null     float64
 6   valorDOC                      24 non-null     float64
 7   valorCartaoCredito            24 non-null     float64
 8   valorCartaoDebito             24 non-null     float64
 9   valorCartaoPrePago            24 non-null     float64
 10  valorTransIntrabancaria       24 non-null     float64
 11  valorConvenios                24 non-null     float64
 12  valorDebitoDireto             24 non-null     float64
 13  valorSa

Aparentemente não temos dados faltantes, mas vamos comprovar:

In [56]:
df.isnull().sum()

datatrimestre                   0
valorPix                        0
valorTED                        0
valorTEC                        0
valorCheque                     0
valorBoleto                     0
valorDOC                        0
valorCartaoCredito              0
valorCartaoDebito               0
valorCartaoPrePago              0
valorTransIntrabancaria         0
valorConvenios                  0
valorDebitoDireto               0
valorSaques                     0
quantidadePix                   0
quantidadeTED                   0
quantidadeTEC                   0
quantidadeCheque                0
quantidadeBoleto                0
quantidadeDOC                   0
quantidadeCartaoCredito         0
quantidadeCartaoDebito          0
quantidadeCartaoPrePago         0
quantidadeTransIntrabancaria    0
quantidadeConvenios             0
quantidadeDebitoDireto          0
quantidadeSaques                0
dtype: int64

Note que data trimestre está como object(string), mas seria mais interessante estar como Date. Vamos comprovar isso.

In [57]:
type(df['datatrimestre'][0])

str

Convertendo para date.

In [58]:
df['datatrimestre'] = pd.to_datetime(df['datatrimestre'])
df['datatrimestre'] = df['datatrimestre'].dt.date

In [59]:
type(df['datatrimestre'][0])

datetime.date

Ajustando as colunas que estão em milhões

In [60]:
monetary_cols = [
    'valorPix', 'valorTED', 'valorTEC', 'valorCheque', 'valorBoleto',
    'valorDOC', 'valorCartaoCredito', 'valorCartaoDebito', 'valorCartaoPrePago',
    'valorTransIntrabancaria', 'valorConvenios', 'valorDebitoDireto', 'valorSaques'
]

for col in monetary_cols:
    df[col] = df[col]*1_000_000
    df[col] = df[col].round(2)

In [61]:
df

datatrimestre      valorPix      valorTED      valorTEC   valorCheque  \
0     2024-12-31  7.714115e+12  1.164727e+13  0.000000e+00  1.874864e+11   
1     2024-09-30  6.975429e+12  1.118022e+13  0.000000e+00  1.943306e+11   
2     2024-06-30  6.284296e+12  1.066256e+13  0.000000e+00  2.215538e+11   
3     2024-03-31  5.429305e+12  9.651626e+12  9.252600e+08  1.904572e+11   
4     2023-12-31  5.300155e+12  1.009092e+13  5.787890e+09  2.214248e+11   
5     2023-09-30  4.469530e+12  9.960252e+12  4.156380e+09  2.181219e+11   
6     2023-06-30  3.900377e+12  1.065746e+13  4.770740e+09  2.284088e+11   
7     2023-03-31  3.447198e+12  9.919603e+12  4.535170e+09  2.406862e+11   
8     2022-12-31  3.342914e+12  1.038212e+13  5.609530e+09  2.422596e+11   
9     2022-09-30  2.936414e+12  1.043776e+13  3.735460e+09  2.670902e+11   
10    2022-06-30  2.543384e+12  1.044957e+13  4.786140e+09  2.710081e+11   
11    2022-03-31  2.067827e+12  9.507158e+12  3.865810e+09  2.554554e+11   
12    2021-12-31  1.916419e+12  9.807293e+12  5.436190e+09  2.642982e+11   
13    2021-09-30  1.556916e+12  9.257046e+12  3.684130e+09  2.711203e+11   
14    2021-06-30  1.105735e+12  8.566787e+12  3.894810e+09  2.649443e+11   
15    2021-03-31  6.250465e+11  7.894414e+12  4.352440e+09  2.535279e+11   
16    2020-12-31  1.498949e+11  8.036705e+12  5.137420e+09  2.782954e+11   
17    2020-09-30  0.000000e+00  7.503982e+12  4.159600e+09  2.593349e+11   
18    2020-06-30  0.000000e+00  6.267004e+12  4.647320e+09  2.378731e+11   
19    2020-03-31  0.000000e+00  6.652135e+12  4.829560e+09  3.188449e+11   
20    2019-12-31  0.000000e+00  6.560043e+12  6.343770e+09  3.726937e+11   
21    2019-09-30  0.000000e+00  6.285701e+12  5.469660e+09  3.857352e+11   
22    2019-06-30  0.000000e+00  5.700740e+12  5.506270e+09  3.885255e+11   
23    2019-03-31  0.000000e+00  5.039381e+12  4.794820e+09  3.759163e+11   

     valorBoleto      valorDOC  valorCartaoCredito  valorCartaoDebito  \
0   2.530071e+12  0.000000e+00        7.286220e+11       2.599298e+11   
1   2.445001e+12  0.000000e+00        6.632884e+11       2.419330e+11   
2   2.360424e+12  0.000000e+00        6.318115e+11       2.391016e+11   
3   2.248973e+12  6.508000e+08        6.074710e+11       2.351116e+11   
4   2.317766e+12  4.684980e+09        6.301913e+11       2.590579e+11   
5   2.296687e+12  1.013189e+10        5.819024e+11       2.408278e+11   
6   2.249060e+12  1.125823e+10        5.619109e+11       2.381722e+11   
7   2.211523e+12  7.467110e+09        5.355204e+11       2.371751e+11   
8   2.259675e+12  9.584630e+09        5.607654e+11       2.641206e+11   
9   2.248417e+12  1.710646e+10        5.219423e+11       2.436329e+11   
10  2.210099e+12  1.708153e+10        5.015147e+11       2.455638e+11   
11  2.032317e+12  1.197636e+10        4.532924e+11       2.301869e+11   
12  2.072054e+12  1.416699e+10        4.679112e+11       2.574583e+11   
13  2.001074e+12  2.230675e+10        3.990541e+11       2.339700e+11   
14  1.839452e+12  2.315884e+10        3.449997e+11       2.143811e+11   
15  1.769760e+12  2.177770e+10        3.104121e+11       2.047529e+11   
16  1.919101e+12  3.870514e+10        3.329872e+11       2.584715e+11   
17  1.959820e+12  5.215684e+10        2.791770e+11       2.287719e+11   
18  1.366623e+12  5.088115e+10        2.243895e+11       1.522468e+11   
19  1.526487e+12  3.062855e+10        2.756591e+11       1.713995e+11   
20  1.579497e+12  3.772879e+10        3.028882e+11       1.936730e+11   
21  1.484836e+12  3.965279e+10        2.724147e+11       1.647356e+11   
22  1.409619e+12  3.471655e+10        2.531614e+11       1.537472e+11   
23  1.468066e+12  3.581280e+10        2.354355e+11       1.517647e+11   

    valorCartaoPrePago  ...  quantidadeCheque  quantidadeBoleto  \
0         8.563790e+10  ...          41511.28        1459164.03   
1         7.853194e+10  ...          44243.73        1466737.62   
2         7.738901e+10  ...          44981.97        1439100.25   
3       

Ajustando as colunas que estão em milhares

In [62]:
0.1 + 0.7

0.7999999999999999

In [63]:
thousands_cols = [
    'quantidadePix', 'quantidadeTED', 'quantidadeTEC', 'quantidadeCheque',
    'quantidadeBoleto', 'quantidadeDOC', 'quantidadeCartaoCredito',
    'quantidadeCartaoDebito', 'quantidadeCartaoPrePago',
    'quantidadeTransIntrabancaria', 'quantidadeConvenios',
    'quantidadeDebitoDireto', 'quantidadeSaques'
]

for col in thousands_cols:
    df[col] = df [col]*1_000



## Estatísticas Descritivas

Nosso conjunto de dados está com muitas colunas, vamos nos concentrar em um tema específico para continuar com a análise. 

Vamos nos concentrar nas séries de Pix e cartões.

In [64]:
date_col = ['datatrimestre']

pix_cols = ['valorPix', 'quantidadePix']

cartao_cols = [
    'valorCartaoCredito', 'quantidadeCartaoCredito',
    'valorCartaoDebito',  'quantidadeCartaoDebito',
    'valorCartaoPrePago', 'quantidadeCartaoPrePago'
]
selected_cols = date_col + pix_cols + cartao_cols
df_pix_cards = df[selected_cols].copy()

In [65]:
df_pix_cards.describe()


valorPix  quantidadePix  valorCartaoCredito  \
count  2.400000e+01   2.400000e+01        2.400000e+01   
mean   2.490207e+12   5.781591e+09        4.448634e+11   
std    2.482513e+12   6.045886e+09        1.568315e+11   
min    0.000000e+00   0.000000e+00        2.243895e+11   
25%    0.000000e+00   0.000000e+00        2.969604e+11   
50%    1.992123e+12   4.043155e+09        4.606018e+11   
75%    4.042665e+12   9.826671e+09        5.669088e+11   
max    7.714115e+12   1.796084e+10        7.286220e+11   

       quantidadeCartaoCredito  valorCartaoDebito  quantidadeCartaoDebito  \
count             2.400000e+01       2.400000e+01            2.400000e+01   
mean              3.566431e+09       2.216744e+11            3.482304e+09   
std               1.080436e+09       3.707392e+10            6.185537e+08   
min               1.907929e+09       1.517647e+11            2.259584e+09   
25%               2.567385e+09       2.019829e+11            2.993007e+09   
50%               3.710315e+09       2.361434e+11            3.684254e+09   
75%               4.472067e+09       2.441156e+11            3.967864e+09   
max               5.351538e+09       2.641206e+11            4.282401e+09   

       valorCartaoPrePago  quantidadeCartaoPrePago  
count        2.400000e+01             2.400000e+01  
mean         4.206254e+10             1.619067e+09  
std          2.831053e+10             1.068768e+09  
min          5.019160e+09             5.884758e+07  
25%          1.334950e+10             7.408515e+08  
50%          4.298120e+10             1.555549e+09  
75%          6.694979e+10             2.489421e+09  
max          8.563790e+10             3.339927e+09

## Estudo das Médias

Nas aulas passadas encontramos o valor:

In [66]:
soma1 = df_pix_cards['valorPix'].sum()
soma2 = df_pix_cards['quantidadePix'].sum()

soma1/soma2

np.float64(430.713035665615)

Por outro lado:

In [67]:
(df_pix_cards['valorPix']/df_pix_cards['quantidadePix']).mean()

np.float64(491.13828791552487)

A diferença existe porque, na segunda abordagem, você está dando peso igual a cada trimestre, independentemente de quantas transações ocorreram naquele período. Já na primeira, cada transação — seja no trimestre que for — “conta” igualmente, o que faz o trimestre com mais transações influenciar mais o resultado final.


| Métrica                                  | Fórmula                                                    | Interpretação                                                  |
| ---------------------------------------- | ---------------------------------------------------------- | -------------------------------------------------------------- |
| Média geral (ponderada)                  | $\sum_i valorPix_i \,/\, \sum_i quantidadePix_i$           | Valor médio por transação considerando todo o volume           |
| Média das médias trimestrais (não-pond.) | $\tfrac{1}{n}\sum_i\bigl(valorPix_i/quantidadePix_i\bigr)$ | Valor médio por transação onde cada trimestre tem “peso” igual |

Use a **média geral** quando quiser o valor médio de toda a amostra, e a **média das médias trimestrais** quando quiser tratar cada período (trimestre) com igual importância.


### Variância e Desvio-Padrão

Esse é o “tamanho típico” da variação em torno de R\$ 491. Na prática, significa que cerca de 68% dos trimestres tiveram valor médio por transação entre

  $$
    491 - 121{,}7 \approx 369 \quad\text{e}\quad 491 + 121{,}7 \approx 613.
  $$

### Interpretação prática

1. **Dispersão moderada**
   Um desvio-padrão de R\$ 122 em torno de R\$ 491 equivale a um coeficiente de variação de

   $$
     \frac{121{,}7}{491} \approx 0{,}25\quad(\,25\%\,),
   $$

   ou seja, as médias trimestrais flutuam em torno de ±25% da média geral.

2. **Potenciais causas**

   * **Sazonalidade**: se alguns trimestres (ex.: fim de ano) tendem a ter transações maiores, isso eleva o desvio.
   * **Mudanças estruturais**: adoção crescente de Pix, promoções ou campanhas podem alterar o ticket médio.

3. **Uso da métrica**

   * **Planejamento**: saber que “normalmente” o ticket médio fica entre R\$ 369 e R\$ 613 ajuda a projetar receitas futuras.
   * **Detecção de anomalias**: trimestres fora desse intervalo (muito acima ou abaixo) podem merecer investigação.

Em suma, embora o valor médio seja R\$ 491, existe uma variabilidade não desprezível de cerca de R\$ 122 entre trimestres, indicando que fatores sazonais ou de comportamento do consumidor fazem o ticket médio de Pix oscilar em torno desse patamar.


## Visualizações

para complementar o processo de análise de dados vamos gerar alguns gráficos, para isso usaremos a biblioteca matplotlib.



**Histograma**
Mostra a distribuição de frequências de uma variável numérica dividida em “bins” (intervalos). Cada barra indica quantos valores caem em cada faixa. Use quando quiser entender a forma geral dos dados (simetria, sesgo, picôs múltiplos) e identificar aglomerados ou lacunas.


In [70]:
import matplotlib.pyplot as plt

plt.hist(x)
plt.show()

NameError: name 'x' is not defined

**Série temporal (line plot)**
Plota um valor ao longo de um eixo temporal conectado por linhas. Ideal para acompanhar tendências, ciclos sazonais e rupturas em séries de tempo (ex.: evolução do ticket médio de Pix trimestre a trimestre). Ajuda a visualizar crescimento, quedas e padrões periódicos.

**Box-plot**
Representa a mediana, quartis e potenciais outliers de um conjunto de observações. A “caixa” vai do primeiro ao terceiro quartil, com uma linha na mediana, e “bigodes” indicando o alcance dos dados sem outliers. Use para comparar dispersão e simetria entre grupos (por exemplo, diferentes canais de pagamento) e para detectar valores atípicos.

**Exercício:** Fazer um estudo das médias semelhante para os diferentes tipos de Cartões.